domain/src/main/java/org/oppia/android/domain/translation/TranslationController.kt

```diff
@@ -164,22 +168,19 @@ class TranslationController @Inject constructor(
    *     result is ever provided). The payload of the data provider is the *previous* selection
```

Comment:
Does this KDoc need to be updated with the new changes? I'm not sure this is actually correct anymore.

Reply:
Updated.